win32ss/gdi/gdi32/CMakeLists.txt

```diff
@@ -57,10 +57,9 @@ set_module_type(gdi32
 
 target_link_libraries(gdi32
     winegdi
-    wine 
+    wine
     win32ksys
     dxguid
-    msvcrtex
```

Comment:
Wait, so this was unnecessary this whole time?

Reply:
looks like it, yes